openadapt/models.py

```diff
@@ -930,6 +929,24 @@ class Replay(db.Base):
     strategy_name = sa.Column(sa.String)
     strategy_args = sa.Column(sa.JSON)
     git_hash = sa.Column(sa.String)
+    logs = sa.orm.relationship("ReplayLog", back_populates="replay")
+
+
+class ReplayLog(db.Base):
```

Comment:
What do you think about renaming "Replay" to "Execution" and "ReplayLog" to "ExecutionLog"?